Gir.toml

```diff
@@ -1,12 +1,21 @@
 [options]
-girs_dir = "gir-files"
+girs_dir = "../gir-files"
```

Comment:
Since we're generating some code now it makes sense to add the usual submodules and makefile.